backend/app/adapter/gqlapi/resolver/authmutation.go

```diff
@@ -67,9 +67,9 @@ func (a AuthMutation) CreateURL(args *CreateURLArgs) (*URL, error) {
 	case url.ErrAliasExist:
 		return nil, ErrURLAliasExist(*customAlias)
 	case url.ErrInvalidLongLink:
-		return nil, ErrInvalidLongLink(u.LongLink)
+		return nil, ErrInvalidLongLink{u.LongLink, err.(url.ErrInvalidLongLink).Violation.String()}
```

Comment:
```suggestion
		return nil, ErrInvalidLongLink{u.LongLink, string(err.(url.ErrInvalidLongLink).Violation)}
```

Reply:
I think this suggestion will work once I change the type of `Violation` from `int` to `string` like you suggested above.

Reply:
Done